dev/beaker/clone_gatling-puppet-load-test.rb

```diff
@@ -0,0 +1,9 @@
+test_name "Clone the gatling-puppet-load-test repo on the target machine"
+
+repo = "gatling-puppet-load-test"
+repo_url = "https://github.com/puppetlabs/#{repo}.git"
+
+hosts.each do |host|
```

Comment:
Yes that's totally right and it should probably *not* work with multiple hosts, unless we can think of a reason it should be. I asked the QA folks and apparently you can define a custom role and use it as a unique identifier for the host. Something like
```yaml
roles:
      - dev_machine
```

```ruby
on(dev_machine, "echo hello")
```
This seems like the way to go, since it will be more explicit. Either that or something like `host = hosts[0]`